resources/schemas.json

```diff
@@ -707,7 +719,17 @@
                         "additionalProperties": {
                             "$ref": "#/components/schemas/TokenView"
                         },
-                        "description": "A map of tokens, indexed by SS58 address of authentication requester"
+                        "description": "A map of tokens, indexed by SS58 address of authentication requester",
+                        "example": {
+                            "5GEpejzbL7Ch8hrRuWsTVCfVRLFZm1zKMF989J4NSiH8JwPn": {
+                                "value": "eyJhbGciOiJIUzM4NCI....",
+                                "expiredOn": "2021-09-17T14:13:06.000Z"
```

Comment:
Maybe we should remove the "Z" to enforce the fact that we do not expose any zone/offset (which is the convention we followed up to now)?